pkg/console/operator/operator.go

```diff
@@ -228,7 +235,7 @@ func (c *ConsoleOperator) sync(_ interface{}) error {
 			}
 		}
 	default:
-		logrus.Printf("Unrecognized version. Desired %s, Actual %s", desiredVersion, currentActualVersion)
+		logrus.Printf("unrecognized version. desired %s, actual %s", desiredVersion, currentActualVersion)
```

Comment:
I understand why we lower-casing the first sentence's first work, but not sure if that should apply to all sentences.